File: LocationMobileApp/app/src/main/java/com/example/location_mobile_app/Location.kt
package com.example.location_mobile_app

import java.time.LocalDateTime
import java.time.LocalDateTime.now
import java.time.format.DateTimeFormatter
import java.util.*

class Location constructor(Latitude: Double, Longitude: Double, TerminalId: String) {
    var id: Int = 0
    var latitude: Double = Latitude
    var longitude: Double = Longitude
    var terminalId: String = TerminalId
    var dateTime: String? = now().format(DateTimeFormatter.ISO_LOCAL_DATE_TIME)

}

File: LocationWebApp/LocationWebApp/Models/Location.cs
using System;

namespace LocationWebApp.Models
{
    public class Location
    {
        public int Id { get; set; }
        public string TerminalId { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public DateTime DateTime { get; set; }
    }
}
